zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/ZeebeTransaction.java

```diff
@@ -60,10 +71,17 @@ public byte[] get(
       final int keyLength)
       throws Exception {
     try {
+      final int keyOffset = 0;
```

Comment:
🤔Interesting so we could at some point use one buffer

Reply:
Yes I believe this is to support wide columns (support for structured data somewhat akin to relational data)